hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/src/main/java/com/google/cloud/bigtable/hbase2_x/replication/HbaseToCloudBigtableReplicationEndpoint.java

```diff
@@ -88,4 +172,7 @@ protected void doStop() {
     cloudBigtableReplicator.stop();
     notifyStopped();
   }
+
+  @Override
+  public void peerConfigUpdated(ReplicationPeerConfig replicationPeerConfig) {}
```

Comment:
What is the default behavior in the ReplicationEndpoint? Please add a TODO to implement it. Hypothetically, we can implement this to enable/disable dry-run mode without deleting the peer.